### ISSUE TITLE
Iliac artery aneurysm

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7158]
**Preferred term label:**
Iliac artery aneurysm
**Synonyms**
Iliac artery dilatation

**Definition (free text, please give PubMed ID)**
like this Abnormal outpouching or sac-like dilatation in an artery that originates from he abdominal aorta. (but note and correct spelling error)

**Parent term (use hpo.jax.org/app)**
Hierarchy needs some additional work in this area

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619472 
PMID: 34010604  2/12

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
